docs-new/docs/contribute/docs/local-building.md

```diff
@@ -1,81 +1,40 @@
 # Build Documentation Locally
 
-## Building the documentation locally
-
-You can run Docsy locally so that you can view the formatted version
-of what you are writing before you push it to github.
-We provide a Docsy run environment in a Docker container,
+You can run MkDocs locally so that you can view the formatted version
+of what you are writing before you push it to GitHub.
+We provide a MkDocs run environment in a Docker container,
 which simplifies the setup
 and makes it easier to upgrade your local build environment
 as the software is updated.
 
-To set up a local Docsy build:
+To set up a local MkDocs build:
 
 1. Install Docker Desktop:
 
-   * [Install on macOS](https://docs.docker.com/desktop/install/mac-install/)
-   * [Install on Linux](https://docs.docker.com/desktop/install/linux-install/)
-   * [Install on Windows](https://docs.docker.com/desktop/install/windows-install/)
+    * [Install on macOS](https://docs.docker.com/desktop/install/mac-install/)
+    * [Install on Linux](https://docs.docker.com/desktop/install/linux-install/)
+    * [Install on Windows](https://docs.docker.com/desktop/install/windows-install/)
 
-1. Execute the following command from the `docs` folder of your clone:
+1. Execute the following command from the root of your clone:
 
-   ```shell
-   make server
-   ```
+     ```shell
+     make docs-serve
+     ```
 
-   It will continue running in its own shell.
+     It will continue running in its own shell.
 
-   > **Note**
-   To utilize the `Makefile`, you must have GNU **make**
-   available on your local machine.
-   Versions are available for all the usual Operating Systems.
+    > **Note**
+    To utilize the `Makefile`, you must have GNU **make**
+    available on your local machine.
+    Versions are available for all the usual Operating Systems.
 
 1. Start contributing!
-Note that Hugo updates the rendered documentation each time you write the file.
+Note that MkDocs updates the rendered documentation each time you write the file.
 
 1. Enter the following in a browser to view the website:
 
-    `http://localhost:1314`
-
-   > **Note**
-   By default, Hugo serves the local docs on port 1313.
-   We have modified that port for the lifecycle-toolkit docs
-   to avoid conflicts with the keptn.github.io docs, which use
-   port 1313 for local builds.
-
-1. Use Ctrl+C to stop the local Hugo server when you are done.
-
-1. To restart the continuous build:
+    `http://localhost:8000`
 
-   * Restart Docker-Desktop, if necessary
-   * If changes have been made to the build tools:
-     * make clone
-     * make build
-   * Run `make server`
-
-### Building markdown files without Hugo
```

Comment:
Why was this deleted? Does MkDocs render the README.md and CONTRIBUTING.md files?  If so, how do I preview those?

Reply:
I don't think this information is particularly valuable for users. Most markdown files are rendered by our docs and the few exceptions are not worth writing and maintaining a whole paragraph I think.